src/@hooks/useAddressConfig.ts

```diff
@@ -67,13 +68,22 @@ export function useAddressConfig(): UseAddressConfig {
     )
   }
 
+  const getVerifiedAddressName = (address: string) => {
```

Comment:
suggestion: adding a return type is recommended for type safety, e.g. `Promise<string|undefined>`